osf/models/node.py

```diff
@@ -1984,12 +1984,22 @@ def use_as_template(self, auth, changes=None, top_level=True):
         )
         new.save()
         # deal with the children of the node, if any
-        for node_relation in self.node_relations.select_related('child').filter(child__is_deleted=False):
-            child = node_relation.child
-            if child.can_view(auth):
-                templated_child = child.use_as_template(auth, changes, top_level=False)
-                NodeRelation.objects.get_or_create(parent=new, child=templated_child,
-                                                   is_node_link=node_relation.is_node_link)
+        for node_relation in self.node_relations.filter(child__is_deleted=False):
+            node_contained = node_relation.child
+            # tempalte child nodes
+            if not node_relation.is_node_link:
+                try:  # Catch the potential PermissionsError above
+                    templated_child = node_contained.use_as_template(auth, changes, top_level=False)
+
+                except PermissionsError:
+                    pass  # If this exception is thrown omit the node from the result set
+                    templated_child = None
+                if templated_child is not None:
```

Comment:
If the only way to hit `templated_child is None` is through a PermissionError, you might consider using try-except-else instead

Reply:
Actually, this can be a pass and no change to templated_child, if you use the except clause, either way works fine though.,